memory/src/main/java/org/opensearch/ml/memory/index/InteractionsIndex.java

```diff
@@ -337,7 +337,10 @@ public void getTraces(String interactionId, int from, int maxResults, ActionList
 
         request.source(searchSourceBuilder);
         request.source().from(from).size(maxResults);
-        request.source().sort(ConversationalIndexConstants.INTERACTIONS_CREATE_TIME_FIELD, SortOrder.ASC);
+        request
+            .source()
+            .sort(ConversationalIndexConstants.INTERACTIONS_CREATE_TIME_FIELD, SortOrder.ASC)
```

Comment:
Do we still need sort by create time if we add sort by trace number ?

Reply:
make sense, trace number should enough, remove create_time sort.